corehq/apps/app_manager/util.py

```diff
@@ -409,7 +409,7 @@ def get_casedb_schema(form):
     This lists all case types and their properties for the given app.
     """
     app = form.get_app()
-    base_case_type = form.get_module().case_type
+    base_case_type = form.get_module().case_type if form.requires_case() else None
```

Comment:
> even if usercase in the form builder turned on, the app properties window doesn't seem to show unless the form requires a case

That might be a bug in [`_form_uses_case`](https://github.com/dimagi/commcare-hq/blob/a59656f1285374f3474eb82bcbfd3e43fd1b981e/corehq/apps/app_manager/views/formdesigner.py#L57), which is used to [enable the data browser](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/views/formdesigner.py#L98-L99). `_form_uses_case` should probably do

```py
return (
    (module and module.case_type and form.requires_case())
    or is_usercase_in_use(domain)
)
```